### ISSUE TITLE
Admin panel: Analysis dashboard bulk downloads

### ISSUE THREAD

**Issue by anisfeld** [prijatelilab/PrijateliTree#152]
## What Issue Are You Seeing or What Feature Needs To Be Added?

We want easy access to the data from the website itself this include high level summaries and the ability to do bulk downloads of pre-processed data.  

We still need to figure out our exact needs--we will start with bulk downloads! (Then, @pepipandi and @anisfeld  will do some analysis and we will refine our request / ask for specific summary tables to be added to the dashboard)

- [x] Create a route called `analysis_dashboard` and a related template. 

- [x] Create a button to download some CSV.
~Idea: Use single endpoint. Button passes data type as parameter. We use a switch to determine what data is pulled.~
- use Buttons built in for dataTable (https://datatables.net/extensions/buttons/).
- Idea: build tables using jinja / python. Display them with dataTables and have csv button to export.

Make the following data downloadable as csv.  

- [x] Session histories ... (session_id, game_id, game types)
     - game type = (network, anonymity, points, is network shown, practice)
     - This information is in the `games` table 

- [x] Answer histories ... (session_id, game_id, student_id, qualtrics_id, position_id, round, answer) 
      - each row is a single answer from a student given a game-round
 
- [ ] #157
     -  each row is a directed edge from ego to alter. 
      
## Are There Any Links That Would Be Helpful?

https://datatables.net/extensions/buttons/

**Comment by anisfeld** [comment]
Easy enough to download csv built entirely in python:

```
return StreamingResponse(
        iter([df.to_csv(index=False)]),
        media_type="text/csv",
        headers={"Content-Disposition": f"attachment; filename=data.csv"},
    )
```

However, making the pandas dataFrame requires code that's redundant -- we already build the tables in jinja to display them (at least this is the case for the denars table) taking advantage of sqlalchemy relations. 

Buttons allow us to download data using off-the-shelf javascript. Examples of buttons very useful:

https://datatables.net/extensions/buttons/examples/

Main issue here is we need to use datatables 2.0.1 instead of 1.7. This might make a headache getting select to work again!

**Comment by anisfeld** [comment]
Closed by #156  / 
Superseded by #157
